### PR TITLE
EPMDJ-7281 Return agent sessions only with active build version, code refactoring

### PR DESCRIPTION
**Pull request by ViktoriiaLebedeva**

### CODE REVIEW

File: core/src/main/kotlin/endpoints/AgentManager.kt [hunk 1]
```diff
@@ -545,54 +537,54 @@ class AgentManager(override val kodein: Kodein) : KodeinAware {
...
-        val agentId = instanceKey.agentKey.agentId
-        updateInstanceStatus(instanceKey, AgentStatus.BUSY).also { instance ->
-            if (instance?.all { it.value.status == AgentStatus.BUSY } == true) {
-                notifyAgents(agentId)
-                logger.debug { "Agent $agentId is busy." }
+    ): Unit = getInstanceState(instanceId)?.let { instanceState ->
+        updateInstanceStatus(instanceId, AgentStatus.BUSY)?.also { instance ->
+            if (instance.all { it.value.status == AgentStatus.BUSY }) {
+                notifyAgents(id)
+                logger.debug { "Agent $id is busy." }
```
Comment (LikhanovMaxim) [inline core/src/main/kotlin/endpoints/AgentManager.kt]: should we add buildVersion to logging for easy investigation of a problem?

Comment (ViktoriiaLebedeva) [reply core/src/main/kotlin/endpoints/AgentManager.kt]: I would like to add instance id to logs, for example to this:
 `DEBUG c.e.d.admin.endpoints.AgentManager - Sending 1 plugins to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sending plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent data of plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent plugin test2code to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Sent plugins [test2code] to agent Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Enabling all plugins for agent with id Petclinic
DEBUG c.e.d.admin.endpoints.AgentManager - Enabling plugin test2code for agent Petclinic...`  
adding buildVersion makes sense too

Comment (ViktoriiaLebedeva) [reply core/src/main/kotlin/endpoints/AgentManager.kt]: There are more changes than I thought. Maybe expand the logs into another task?

File: core/src/main/kotlin/service/RequestValidator.kt [hunk 2]
```diff
@@ -46,8 +47,10 @@ internal class RequestValidator(override val kodein: Kodein) : KodeinAware {
 
 
 
+
```
Comment (Mazel-Tovr) [inline core/src/main/kotlin/service/RequestValidator.kt:50]: remove line pls

**Pull request closed without merging**